eng/Version.Details.xml

```diff
@@ -49,6 +49,10 @@
       <Sha>27af96bdb7ba8d6d7ea9ad53fc76cd1d1aa80703</Sha>
       <RepoName>nuget-client</RepoName>
     </Dependency>
+    <Dependency Name="Microsoft.Extensions.Logging" Version="3.0.0-rtm.19279.9">
```

Comment:
Where did this version number come from? The latest build should be "3.0.0-preview6.19279.7"

Reply:
Fixed this.